phase1-coordinator/src/objects/round.rs

```diff
@@ -145,26 +145,21 @@ impl Round {
                 Chunk::new(
                     chunk_id as u64,
                     verifier.clone(),
-                    storage
-                        .to_path(&Locator::ContributionFile(ContributionLocator::new(
-                            round_height,
-                            chunk_id as u64,
-                            0,
-                            true,
-                        )))
-                        .expect("failed to create locator path"),
-                    storage
-                        .to_path(&Locator::ContributionFileSignature(ContributionSignatureLocator::new(
-                            round_height,
-                            chunk_id as u64,
-                            0,
-                            true,
-                        )))
-                        .expect("failed to create locator path"),
+                    storage.to_path(&Locator::ContributionFile(ContributionLocator::new(
+                        round_height,
+                        chunk_id as u64,
+                        0,
+                        true,
+                    )))?,
+                    storage.to_path(&Locator::ContributionFileSignature(ContributionSignatureLocator::new(
+                        round_height,
+                        chunk_id as u64,
+                        0,
+                        true,
+                    )))?,
```

Comment:
it's good to have error propagation instead of using `.expect`

Reply:
yes trying to eliminate these hidden expects one by one :smile: